github/ci/prow/files/plugins.yaml

```diff
@@ -124,6 +147,7 @@ approve:
   - kubevirt/kubevirt-tutorial
   - kubevirt/kubevirt
   - kubevirt/containerized-data-importer
+  - nmstate/kubernetes-nmstate
```

Comment:
If you want to enable tide, you will also have to add it to the `tide` section in `config.yaml`.

Reply:
+1 on adding tide https://github.com/kubernetes/test-infra/tree/master/prow/tide

Reply:
Added it to Tide with `squash` as merge policy